### ISSUE TITLE
will this tool can still use with catalina and gtx 1650 ?

### ISSUE THREAD

**Issue by LeXuanKhanh** [corpnewt/Web-Driver-Toolkit#10]
will this tool can still use with catalina ? im using gtx 1650 laptop when install the patched driver it said failed

**Comment by xGapples** [comment]
No, this does not patch the drivers to enable hardware acceleration in macOS Mojave / Catalina.

Regardless of this, the 1650 is not supported by the web drivers in the first place
